inference-engine/src/vpu/graph_transformer/src/stages/broadcast.cpp

```diff
@@ -143,12 +141,12 @@ void FrontEnd::parseBroadcast(
                      "{} layer with name {} and explicit mode must have 1D target shape tensor, "
                      "actually provided {}D tensor",
                      layer->type, layer->name, shapeDesc.numDims());
-    VPU_THROW_UNLESS(shapeDim == output->desc().numDims(),
+    VPU_THROW_UNLESS(shapeDim == output->desc().numDims() || modeString != "explicit",
                      "{} layer with name {} and explicit mode must have target shape tensor with "
                      "size equals to number of output dims, expected [{}], provided [{}]",
                      layer->type, layer->name, output->desc().numDims(), shapeDim);
 
-    const auto mode = modeString == "numpy" ? BroadcastMode::NUMPY : BroadcastMode::EXPLICIT;
+    const auto mode = modeString == "numpy" ? BroadcastMode::NUMPY : (modeString == "bidirectional" ? BroadcastMode::BIDIRECTIONAL : BroadcastMode::EXPLICIT);
```

Comment:
Nitpick: you can create a static const map of string mode and BroadcastMode enum value matches.
BTW, in this case, you can convert strings to BroadcastMode at the beginning of parsing

Reply:
I've changed it, can you please take a look?